### DISCUSSION TITLE
Instant command help for vim?

### DISCUSSION THREAD

**Discussion by x86dev** [helix-editor/helix#1345] [General] [answered]
I bare use(d) vim and only know some very basic commands to do some rough editing on servers where I really need to.

One of the coolest features of Helix so far IMHO is that instant command help when you enter the command mode (via ":") plus its tab support for it.

Does anybody with more vim background know if there's anything similar for vim? That would immensely help when having to use vim (and not using / remembering all those commands all day long).

Anyway, keep up the good work, looking forward to the next releases!

**Comment by sudormrfbin** [comment, marked as answer]
You can set the [`wildmenu`](https://vi.stackexchange.com/questions/11411/is-there-a-version-of-vim-or-plugin-with-a-command-mode-auto-completion) option. Note that it doesn't show documentation like helix; you probably need a plugin for that (not sure if it even exists). And if you ssh into other machines regularly then installing a plugin might be hard.

**Comment by x86dev** [comment]
Great, never heard of that yet, thanks!
